youtube_dl/extractor/jamendo.py

```diff
@@ -80,7 +93,7 @@ def _real_extract(self, url):
         thumbnail = self._html_search_meta(
             'image', webpage, 'thumbnail', fatal=False)
         duration = parse_duration(self._search_regex(
-            r'<span[^>]+itemprop=["\']duration["\'][^>]+content=["\'](.+?)["\']',
+            r'data-bundled-models=["\']{[^}]+{[^{*}]+duration&quot;:(.+?),',
```

Comment:
No, parse as JSON. Or from ld+json.

Reply:
Thank you for your review ! I've tried to do that, however I guess that Jamendo has an issue with the song duration in the ld+json part. Time format for the duration is ISO8601 I guess, but for one song lasting 210s, it is written as `PT0.21S` whereas for another lasting 141s, it is `PT0.141S`. Thus, parsing the Json would give incoherent results I guess. Moreover, this format can't be parsed by `utils.parse8601()`, so I don't really know what to do.